design/ceph/object/ceph-cosi-driver.md

```diff
@@ -0,0 +1,150 @@
+# Ceph COSI Driver Support
+
+## Targeted for v1.12
+
+## Background
+
+Container Object Storage Interface (COSI) is a specification for container orchestration frameworks to manage object storage. Even though there is no standard protocol defined for Object Store, it has flexibility to add support for all. The COSI spec abstracts common storage features such as create/delete buckets, grant/revoke access to buckets, attach/detach buckets, and more. COSI released v1alpha1 with Kubernetes 1.25. COSI is a new project and is not yet fully integrated by Kubernetes.
```

Comment:
> Even though there is no standard protocol defined for Object Store, it has flexibility to add support for all. 

Not sure what this means, can you reword it?

> COSI is a new project and is not yet fully integrated by Kubernetes.

Is this an old statement? It's fully implemented now for K8s, right?
```

Reply:
It is alpha status and it is not integrated with K8s like CSI